activation-registry/src/main/javadoc/doc-files/speclicense.html

```diff
@@ -0,0 +1,72 @@
+<html>
```

Comment:
this file does not belong to implementations; remove it

Reply:
Removed